temperature/models.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
+
+from sqlalchemy import Column, Integer, ForeignKey, DateTime, Float
+from database import Base
```

Comment:
Ensure that the `Base` class is correctly imported from the `database` module, as it serves as the declarative base for the ORM models.